net-im/coyim/Makefile

```diff
@@ -6,7 +6,7 @@ DISTVERSION=	0.3.11
 CATEGORIES=	net-im
 
 MAINTAINER=	santhosh.raju@gmail.com
-COMMENT=	Safe and secure chat client
+COMMENT=	A safe and secure by default chat client
```

Comment:
Due to the rules about COMMENT in FreeBSD port, a comment should not start with "A" / "An" or "The".

https://www.freebsd.org/doc/en/books/porters-handbook/book.html#makefile-comment

Reply:
Solved ;)

Reply:
LGTM, pulling in the changes.

Thank you.